x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -95,9 +95,17 @@ export const MisconfigurationFindingsDetailsTable = memo(
 
     const [currentFilter, setCurrentFilter] = useState<string>('');
 
+    const [sortField, setSortField] = useState<
+      MISCONFIGURATION.RESULT_EVALUATION | MISCONFIGURATION.RULE_NAME | 'resource' | 'rule'
+    >(MISCONFIGURATION.RESULT_EVALUATION);
+    const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>('asc');
```

Comment:
So each field is ASC. Do we show the most severe statuses to the user by default?

Reply:
so we actually sort this alphabetically even when we set it for Severity (we have the same thing on Findings table too I believe)